__tests__/mockClient.test.js

```diff
@@ -0,0 +1,96 @@
+const Q = require('q');
+const request = Q.denodeify(require('request'));
+const MockClient = require('../src/client/Client');
+const MOCKSERVER_URI = 'localhost:3000';
+
+describe('when adding routes on the same uri', () => {
+	let mockClient;
+	beforeEach(() => {
+		mockClient = new MockClient(3030, MOCKSERVER_URI);
+	});
+
+	it('should return first response', done => {
```

Comment:
To be more precise, I would prefer describing it as `should return first response when route is added once`, or something similar with `... when ...`.